packages/cli/src/executions/execution.service.ts

```diff
@@ -146,7 +146,7 @@ export class ExecutionService {
 		if (!execution.data.executionData) throw new AbortedExecutionRetryError();
 
 		if (execution.finished) {
-			throw new ApplicationError('The execution succeeded, so it cannot be retried.');
+			throw new UnexpectedError('The execution succeeded, so it cannot be retried.');
```

Comment:
This is transient? So OperationalError?

Reply:
This error is part of a workaround for the root issue that we should not be sending requests to retry an execution with `success` status.